enumerable.rb

```diff
@@ -0,0 +1,31 @@
+module Enumerable
+  def my_all?
+    return unless block_given?
+
+    response = false
+    @list.each do |n|
+      response = true if yield(n)
+    end
```

Comment:
- [ ] good job on the enumerables 👍 however according to the requirements the MyList class should implement the `#each` method that yields all members of @list in succession. this means that you should write that function yourself.